test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -1655,4 +1821,92 @@ public void queueOrderInTxnContext() throws Exception {
             assertThat(order).isEqualTo(validator.get(i));
         }
     }
+
+    // ****** Utility Functions *******
+
+    private Uuid getUuid() {
+        UUID id = UUID.randomUUID();
+        return Uuid.newBuilder()
+                .setMsb(id.getMostSignificantBits())
+                .setLsb(id.getLeastSignificantBits())
+                .build();
+    }
+
+    private List<ExampleSchemas.Contact> getContactsWithPhone(String commonMobile) {
+        // Phone Number common across all three ContactBooks
+        final ExampleSchemas.PhoneNumber phoneNumberCommon = ExampleSchemas.PhoneNumber.newBuilder()
+                .setHome("352-546-890")
+                .addMobile("611-9458741")
+                .build();
+
+        final ExampleSchemas.PhoneNumber phoneNumberWorkBook = ExampleSchemas.PhoneNumber.newBuilder()
+                .addMobile(commonMobile)
```

Comment:
why do we not set this in the friends book?

Reply:
Because this is an example of a phone exclusive to a contact in the work book..